### ISSUE TITLE
Dragging non-files over the ngf-drop should not turn on valid dragover class

### ISSUE THREAD

**Issue by fracz** [danialfarid/ng-file-upload#2007]
If you drag a non-file over the `ngf-drop`, it should not turn on valid dragover class. IMO, it should set the rejection CSS class or does not react to such drag at all.

Here is what I mean (Chrome):

![dragging-non-files](https://user-images.githubusercontent.com/3918865/30589029-41914b18-9d39-11e7-9575-b4b36c7e0198.gif)

**Comment by eddieconk** [comment]
Agreed, this also causes issues with dragging and dropping text within an input element. By not catching this case, dragging and dropping text is disabled.
